deploy/service.yaml

```diff
@@ -198,7 +191,7 @@ spec:
     spec:
       containers:
         - name: prometheus-service
-          image: keptncontrib/prometheus-service:0.4.0
+          image: imrajdas/prometheus-service:ci
```

Comment:
Please change it to: `keptncontrib/prometheus-service:latest`

Reply:
Done